packages/repository/src/types/buffer.ts

```diff
@@ -4,40 +4,42 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import * as util from 'util';
-import {Type, Any} from './type';
+import {Type} from './type';
 import {Options} from '../common-types';
 
+// tslint:disable:no-any
+
 /**
  * Buffer (binary) type
  */
 export class BufferType implements Type<Buffer> {
   readonly name = 'buffer';
 
-  isInstance(value: Any) {
+  isInstance(value: any) {
     return value == null || Buffer.isBuffer(value);
   }
 
   defaultValue() {
     return Buffer.from([]);
   }
 
-  isCoercible(value: Any): boolean {
+  isCoercible(value: any): boolean {
     if (value == null) return true;
     if (typeof value === 'string') return true;
     if (Buffer.isBuffer(value)) return true;
     if (Array.isArray(value)) return true;
     return false;
   }
 
-  coerce(value: Any, options?: Options) {
+  coerce(value: any, options?: Options) {
     if (value == null) return value;
     if (Buffer.isBuffer(value)) return value as Buffer;
     if (typeof value === 'string') {
       options = options || {};
       const encoding = options.encoding || 'utf-8';
-      return Buffer.from(value as string, encoding);
+      return Buffer.from(value, encoding);
     } else if (Array.isArray(value)) {
-      return Buffer.from(value as Array<Any>);
+      return Buffer.from(value);
```

Comment:
TS already knows that `value` is an array because of the `typeof` check above, no need to cast the value again.